docs/source/building_applications/rag.md

```diff
@@ -82,10 +81,14 @@ results = client.tool_runtime.rag_tool.query(
 One of the most powerful patterns is combining agents with RAG capabilities. Here's a complete example:
 
 ```python
+from llama_stack_client.types.agent_create_params import AgentConfig
+from llama_stack_client.lib.agents.agent import Agent
+
 # Configure agent with memory
 agent_config = AgentConfig(
-    model="Llama3.2-3B-Instruct",
+    model="meta-llama/Llama-3.2-3B-Instruct",
     instructions="You are a helpful assistant",
+    enable_session_persistence=True,
```

Comment:
was this necessary? this should not be necessary for the code to work right?

Reply:
interesting that's good feedback. could you set it to `False` please for this doc?

Reply:
sure 😃

Reply:
I am fixing this in https://github.com/meta-llama/llama-stack/pull/1012

Reply:
@terrytangyuan is it worth removing from the example code then?

Reply:
Well the example doesn't work for the current released version so it might be nice to set it to false here so it works across versions